.gitignore

```diff
@@ -82,4 +82,5 @@ build-harness/
 !vbh/.build-harness-vendorized
 !vbh/build-harness
 !vbh/build-harness-extensions
+testbin/
```

Comment:
What does `testbin/` come from?

Reply:
When I run `make unit-test` locally it generates this directory with some files.